express/tsconfig.json

```diff
@@ -25,8 +26,8 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "commonjs",                                /* Specify what module code is generated. */
-    "rootDir": "src",                                    /* Specify the root folder within your source files. */
+    "module": "commonjs" /* Specify what module code is generated. */,
```

Comment:
esm modules pls

Reply:
I set NodeNext modules. because when I set es modules dotenv wasn't working. 
Can you suggest what the problem might be